features/step_definitions/mobile_steps.rb

```diff
@@ -33,3 +33,7 @@
 When /^I open the drawer$/ do
   find('#menu_badge').click
 end
+
+Then /^the aspect dropdown should be labeled "([^"]*)"/ do |label|
+  expect(page).to have_css("#user_aspects option.list_cover", :text => label)
```

Comment:
That ends up here https://github.com/diaspora/diaspora/blob/develop/features/step_definitions/web_steps.rb#L75

Let's check [the docs](http://www.rubydoc.info/gems/capybara/Capybara/Node/Actions#select-instance_method), so we can select it by it's ID, name and label text. ID falls short, that needs to be unique inside the entire page, leaving us with name and label text. We have no label, so we're left with name. When we look at the step definition again, we can notice that it allows us to scope the lookup into a parent container. Seems like a reasonable alternative to make this fully reusable.